x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/ResponseValueUtils.java

```diff
@@ -144,7 +146,7 @@ private static Object valueAt(DataType dataType, Block block, int offset, BytesR
                     throw new UncheckedIOException(e);
                 }
             }
-            case SHORT, BYTE, FLOAT, HALF_FLOAT, SCALED_FLOAT, OBJECT, DATE_PERIOD, TIME_DURATION, DOC_DATA_TYPE, TSID_DATA_TYPE, NULL,
+            case SHORT, BYTE, HALF_FLOAT, SCALED_FLOAT, OBJECT, DATE_PERIOD, TIME_DURATION, DOC_DATA_TYPE, TSID_DATA_TYPE, NULL,
```

Comment:
If we're no longer using floats for score, then the changes to ResponseValueUtils.java can be completely removed. Although, they are somewhat benign, just not strictly necessary at this point.